Add_User.py

```diff
@@ -0,0 +1,13 @@
+def Add_User():
```

Comment:
Maybe it is a better choice to re-structure the code in something like this:
- application.py (our main program)
- utils.py (library with all the functions add_user(), check_address(), etc)
Also, maybe it is worth to have a look to best practices for python: https://github.com/naming-convention/naming-convention-guides/tree/master/python
(filenames are written in lower case / lower_with_under)

Reply:
I will try this